### PR TITLE
Better i18n extraction

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: _misc/gather_i18n.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+from __future__ import unicode_literals
+
+import argparse
+import logging
+import click
```
Comment (Pikkupomo) [inline _misc/gather_i18n.py]: I think this is missing from requirements

**Comment by Pikkupomo** [comment]
When the missing requirement is fixed, LGTM and seems to extract alot of stuff

File: _misc/gather_i18n.py (hunk 1 above)
Comment (akx) [inline _misc/gather_i18n.py]: It's only a requirement for these development CLI tools. You can install it by hand, `pip install click` will do.

**Comment by akx** [comment]
> alot of stuff

**Comment by akx** [comment]
@Pikkupomo : removed the dependency on Click altogether.

**Pull request merged**